packages/react-components/react-utilities/src/hooks/useAnimationFrame.ts

```diff
@@ -9,6 +10,11 @@ import { useBrowserTimer } from './useBrowserTimer';
  * @returns A pair of [requestAnimationFrame, cancelAnimationFrame] that are stable between renders.
  */
 export function useAnimationFrame() {
+  const isDOM = canUseDOM();
+
   // TODO: figure it out a way to not call global.requestAnimationFrame and instead infer window from some context
-  return useBrowserTimer(requestAnimationFrame, cancelAnimationFrame);
+  const setAnimationFrame = isDOM ? requestAnimationFrame : setTimeout;
+  const clearAnimationFrame = isDOM ? cancelAnimationFrame : clearTimeout;
```

Comment:
I am more curious why it should do anything on a server. Would you wind to add a story to `apps/ssr-tests-v9` with usage of this hook?

Reply:
We don't know who will use the hook, and as `requestAnimationFrame` is not available on SSR and creates async code on the browser, to keep consistent to what we can expect from it we can call `setTimeout` instead to mimic the same behavior.

I've added a story test to cover that scenario.

Reply:
So, why not `const setAnimationFrame = isDOM ? requestAnimationFrame : noop;`?

I believe that the result for a server, will be the same.

Reply:
Makes sense. Changed.